src/basic/macro.h

```diff
@@ -29,6 +29,19 @@
 #define _alignptr_ __attribute__((__aligned__(sizeof(void*))))
 #define _warn_unused_result_ __attribute__((__warn_unused_result__))
 
+#if defined(BUILT_WITH_COVERAGE) && BUILT_WITH_COVERAGE
+/* We need to explicitly call __gcov_dump() in places where we use _exit(), since
+ * _exit() skips at-exit hooks resulting in lost coverage */
+#  include <unistd.h>
+extern void __gcov_dump(void);
+
+_unused_ _noreturn_ static void _coverage__exit(int status) {
```

Comment:
please also make this inline, otherwise it shouldn't be in a header file.

And once you make it `inline` you can drop the `_unused_`, because you don't need it anymore

Reply:
Indeed, thanks!